lib/tasks/usertasks.rake

```diff
@@ -0,0 +1,45 @@
+
+# usertasks.rake
```

Comment:
@amarkpark thoughts on adding a note how to use these scripts on the `README.md` file?